### PR TITLE
GitHub issue template with repros via GitHub Actions

### PR DESCRIPTION
**Pull request by cspotcode**
Related: #492

I had an idea to facilitate repros via Github Actions.

I setup a repository with Github Actions set to run a shell script.  The idea is you can open a PR on that repository, modifying the code however you need to reproduce an issue.  Github Actions will automatically run the reproduction.

One benefit is that reporters don't need to create their own git repository to post a reproduction.  We offer a preconfigured repo for them to fork, paste their code, and push.  Or for really simple cases, people might even be able to post their repro using Github's web UI.

https://github.com/cspotcode/ts-node-repros

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### Minimal reproduction
+
+<!--
+
+This link explains why we ask for a minimal reproduction.  Thank you in advance!
+https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff
+
+You can create a reproduction here:
+https://github.com/cspotcode/ts-node-repros
```
Comment (blakeembrey) [inline .github/ISSUE_TEMPLATE.md]: Do you want to move this under TypeStrong too?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+### Steps to reproduce the problem
+
+
+
```
Comment (blakeembrey) [inline .github/ISSUE_TEMPLATE.md]: Nit: Let's use the same three white spaces for each section?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (cspotcode) [reply .github/ISSUE_TEMPLATE.md]: Yeah, that'd be ideal.

Comment (cspotcode) [reply .github/ISSUE_TEMPLATE.md]: @blakeembrey what's the best way to do this?  Push the code to TypeStrong/ts-node-repros?  Or use Github's "Transfer Ownership" button.

**Comment by cspotcode** [comment]
I tried to transfer the repository to TypeStrong, but Github showed me this message: "You don’t have the permission to create public repositories on TypeStrong"

**Comment by blakeembrey** [comment]
@cspotcode Invited you to the org, sorry for the delay. That _should_ let you transfer.

**Comment by cspotcode** [comment]
@blakeembrey Thanks!  Transferred and merged.

**Pull request merged**